src/server/scripts/Northrend/zone_zuldrak.cpp

```diff
@@ -1017,7 +1017,7 @@ class spell_scourge_disguise_expiring : public AuraScript
     {
         if (Player* player = GetTarget()->ToPlayer())
             if (Aura* aura = player->GetAura(SPELL_SCOURGE_DISGUISE))
-                player->Talk(scourge_disguise_expiring_warning, CHAT_MSG_RAID_BOSS_WHISPER, LANG_UNIVERSAL, 0, nullptr);
+                player->Talk(TEXT_DISGUISE_WARNING, CHAT_MSG_RAID_BOSS_WHISPER, 0, nullptr);
```

Comment:
give a look to https://github.com/TrinityCore/TrinityCore/commit/5f541491074765a346c7d0a83edb5f1fbf9240b1 maybe it's better way.

Reply:
I didn't become any wiser by reading that, `Unit::Talk` and `Battleground::SendBroadcastText` do roughly the same except for how they pick targets.

Reply:
I am not sure how your link should help him @Aokromes

Reply:
IMHO those battleground broadcast examples should be better suited for targeting multiple players.
Would it be helpful to look at existing code where `CHAT_MSG_RAID_BOSS_WHISPER` is used already?
https://github.com/TrinityCore/TrinityCore/blob/f1a1acdc89a6521fddca98cc8ef8d6e7ed0b9495/src/server/game/Texts/CreatureTextMgr.cpp#L324-L335